docs/core/entity/sensor.md

```diff
@@ -38,6 +38,7 @@ If specifying a device class, your sensor entity will need to also return the co
 | `SensorDeviceClass.BATTERY` | % | Percentage of battery that is left
 | `SensorDeviceClass.CO2` | ppm | Concentration of carbon dioxide.
 | `SensorDeviceClass.CO` | ppm | Concentration of carbon monoxide.
+| `SensorDeviceClass.CONDUCTIVITY` | S/cm, mS/cm, µS/cm | Conductivity.
```

Comment:
Please also update the page for the number entity. Thanks!

https://developers.home-assistant.io/docs/core/entity/number#available-device-classes